zombies.js

```diff
@@ -25,21 +28,28 @@
  * @property {number} damage
  */
 
+function Weapon(name, damage) {
+  this.damage = damage;
+  Item.call(this, name);
+}
+Weapon.prototype = Object.create(Item.prototype, {
+  constructor: {
+    value: Item
+  }
+});
```

Comment:
This should follow structure: `DeathMetalSong.prototype = Object.create(MetalSong.prototype);`